config.go

```diff
@@ -375,6 +375,33 @@ func (c *CascadingPaginationColumnConfig) FallbackPaginationColumnName() (string
 	return c.FallbackColumn, true
 }
 
+type DataIterationBatchSizePerTableOverride struct {
+	// Lower limit for rowLength, if a rowLength <= MinAvgRowLength, ControlPoints[MinAvgRowLength] will be used
+	MinAvgRowLength int
+	// Upper limit for rowLength, if a rowLength >= MaxAvgRowLength, ControlPoints[MaxAvgRowLength] will be used
+	MaxAvgRowLength int
+	// Map of rowLength => batchSize used to calculate batchSize for new rowLengths, results stored in TableOverride
+	ControlPoints map[int]uint64
+	// Map of schemaName(source schema) => tableName => batchSize to override default values for certain tables
+	TableOverride map[string]map[string]uint64
+}
+
+func (d *DataIterationBatchSizePerTableOverride) Validate() error {
+	if d != nil {
+		if _, found := d.ControlPoints[d.MinAvgRowLength]; !found {
+			return fmt.Errorf("must provide batch size for MinAvgRowLength")
+		}
+		if _, found := d.ControlPoints[d.MaxAvgRowLength]; !found {
+			return fmt.Errorf("must provide batch size for MaxAvgRowLength")
+		}
```

Comment:
Should we set `MinAvgRowLength` and `MaxAvgRowLength` to have defaults values as well?